linux/guest_customization/check_hostname_and_domain.yml

```diff
@@ -27,14 +29,18 @@
     dest_path: "{{ current_test_log_folder }}/hostname_f.txt"
 
 - name: "Get guest FQDN"
-  command: cat "{{ current_test_log_folder }}/hostname_f.txt"
-  changed_when: False
+  command: "cat {{ current_test_log_folder }}/hostname_f.txt"
+  failed_when: False
```

Comment:
"changed_when: False" is added after result from ansible-lint.

Reply:
Why to set changed_when=False?

Reply:
When we run "ansible-lint" on playbooks there are warning like this:
```
[301] Commands should not change things if nothing needs doing
```
So for shell or command task if it will not change anything only display or get something, then we can add this "changed_when: false".